pkg/kubectl/generate/versioned/priorityclass.go

```diff
@@ -69,7 +71,10 @@ func (s PriorityClassV1Generator) Generate(params map[string]interface{}) (runti
 	if !found {
 		return nil, fmt.Errorf("expected string, found %v", description)
 	}
-	delegate := &PriorityClassV1Generator{Name: name, Value: value, GlobalDefault: globalDefault, Description: description}
+
+	preemptionPolicy := apiv1.PreemptionPolicy(params["preemptionPolicy"].(string))
```

Comment:
you didn't add to `ParamNames()` or set up help in `create_priorityclass.go` or register a flag, so I don't think this can actually even be set. Also, prefer `preemption-policy` over camelcase for flags (see `global-default` example)